packages/editor/src/components/block-draggable/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { Draggable } from '@wordpress/components';
+
+const BlockDraggable = ( { children, clientId, rootClientId, blockElementId, layout, order, onDragStart, onDragEnd } ) => {
```

Comment:
The vision for this component is that we should only provide one prop that identifies the block to be dragged. The rest (except for the optional `onDragStart` and `onDragEnd` event handlers) should be derived or retrieved from the state.

I wonder whether at this stage this is useful or just introduces more indirection with no real benefits. We may want to remove it completely in favor of `Draggable` at the moment.

Reply:
Agreed, it doesn't add a lot of value at the moment especially since it's used only once but I have a small preference for keeping it. We can make it more useful. For instance the `rootClientId` and `order` props can already be added using `withSelect` and avoid passing them from above.